### PR TITLE
Favorite topics delete

### PR DESCRIPTION
**Pull request by khamzahabibullah**
Added delete action to favorite topics
To test go to my topics and try deleting a bookmark.

### CODE REVIEW

File: app/assets/stylesheets/components/_footer.scss [hunk 1]
```diff
@@ -1,14 +1,15 @@
-
 .footer {
-  background: #F4F4F4;
+  background: #f4f4f4;
```
Comment (andrerferrer) [inline app/assets/stylesheets/components/_footer.scss:2]: I'd rather use the `$light-gray`as defined on the [config](https://github.com/Sajakhtar/cryptocanons/blob/master/app/assets/stylesheets/config/_colors.scss)

@khamzahabibullah 
@Sajakhtar

File: app/assets/stylesheets/components/_footer.scss [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
   display: flex;
   align-items: center;
   justify-content: space-between;
   height: 64px;
   padding: 0px 15px;
-  color: rgba(0,0,0,0.3);
+  color: rgba(0, 0, 0, 0.3);
   // flex-wrap: wrap;
   font-size: 0.8rem;
+  position: absolute;
```
Comment (andrerferrer) [inline app/assets/stylesheets/components/_footer.scss:11]: avoid position absolute as much as possible.

Comment (andrerferrer) [reply app/assets/stylesheets/components/_footer.scss:11]: I teach 3 ways of sticking the footer to the bottom without position absolute [here](https://github.com/andrerferrer/quickTips/blob/master/CSS/Footer.md)

@khamzahabibullah @Sajakhtar

File: app/assets/stylesheets/components/_footer.scss [hunk 3]
```diff
@@ -25,7 +26,7 @@
   opacity: 1;
 }
 .footer .fa-heart {
-  color: #D23333;
+  color: #d23333;
```
Comment (andrerferrer) [inline app/assets/stylesheets/components/_footer.scss:29]: same as above.
Prefer using SCSS variables like `$light-gray` for consistency around the app and reusability

File: app/views/favorite_topics/index.html.erb [hunk 4]
```diff
@@ -1 +1,11 @@
-<h1>Favortie Topics</h1>
+<div class="container">
+  <h1>Favortie Topics</h1>
```
Comment (andrerferrer) [inline app/views/favorite_topics/index.html.erb:2]: Typo here

File: app/controllers/favorite_topics_controller.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 class FavoriteTopicsController < ApplicationController
   def index
-    @favorite_topic = FavoriteTopic.all
+    @favorite_topics = FavoriteTopic.all
```
Comment (andrerferrer) [inline app/controllers/favorite_topics_controller.rb:3]: good catch

File: config/routes.rb [hunk 6]
```diff
@@ -6,6 +6,5 @@
     resources :favorite_topics, only: :create
   end
 
-  resources :favorite_topics, only: [:index]
-
+  resources :favorite_topics, only: [:index, :destroy]
```
Comment (andrerferrer) [inline config/routes.rb:9]: neat routes. Very good.

**Review by andrerferrer — COMMENTED** [review]
Please check the comments
@Sajakhtar @khamzahabibullah

File: app/views/favorite_topics/index.html.erb (hunk 4 above)
Comment (Sajakhtar) [reply app/views/favorite_topics/index.html.erb:2]: this  typo should be resolved in the next merge

File: app/assets/stylesheets/components/_footer.scss (hunk 1 above)
Comment (Leprof20) [reply app/assets/stylesheets/components/_footer.scss:2]: Done, thanks!

File: app/assets/stylesheets/components/_footer.scss (hunk 2 above)
Comment (Leprof20) [reply app/assets/stylesheets/components/_footer.scss:11]: Thanks. Done!

File: app/assets/stylesheets/components/_footer.scss (hunk 3 above)
Comment (Leprof20) [reply app/assets/stylesheets/components/_footer.scss:29]: Thanks, Done!

**Pull request merged**